src/App/App.jsx

```diff
@@ -0,0 +1,121 @@
+import CountdownScreen from '../pages/CountdownScreen'
+import StartScreen from '../pages/StartScreen'
+import { useState } from 'react'
+import { Route, Switch, useHistory } from 'react-router-dom'
+
+function App() {
+  const { push } = useHistory()
+  const DURATIONTWENTYFIVE = { minutes: 25, seconds: 0 }
+  const DURATIONFIFTY = { minutes: 50, seconds: 0 }
+
+  const [isDurationLong, setIsDurationLong] = useState(false)
+  const [isActive, setIsActive] = useState(false)
+  const [isPaused, setIsPaused] = useState(false)
+  const [isTimerExpired, setIsTimerExpired] = useState(false)
+  const [[endHours, endMinutes], setEndTime] = useState([])
+  const [[countdownMinutes, countdownSeconds], setCounter] = useState([
+    DURATIONTWENTYFIVE.minutes,
+    DURATIONTWENTYFIVE.seconds,
+  ])
+
+  function handleDurationShort() {
```

Comment:
Just a suggestion and personal preference; you could think about moving all the functions below your `return` statement. This way we can immediately see what the `App.js` returns and hide complex code. This would make it easier to read the code. BUT this is a personal preference and the way you did it here is totally fine 🙂